.gitignore

```diff
@@ -4,4 +4,4 @@ target/
 .classpath
 .settings/
 /.metadata/
-.recommenders/
```

Comment:
I don't understand why the `.flattened-pom.xml` files were added to the PR. the first line in `.gitignore` is supposed to exclude them.

Reply:
We're removing them, because any changes would be ignored, since they are in `.gitignore`